packages/editor/src/editor.spec.ts

```diff
@@ -199,6 +218,50 @@ describe('Editor', () => {
     expect(content).to.be.equal('1 + 1');
   });
 
-  // TODO: e2e or mock tests for _getExtension
-  // TODO: Write a node script that acts as an external editor and write e2e tests
+  context('runEditCommand', () => {
+    context('when editor is not defined', () => {
+      before(() => {
+        cmd = null;
+        delete process.env.EDITOR;
```

Comment:
You can drop this, it’s already being taken care of by the same line in the outer `beforeEach` hook